common/persistence/nosql/nosqlplugin/cassandra/db.go

```diff
@@ -40,15 +40,38 @@ type cdb struct {
 
 var _ nosqlplugin.DB = (*cdb)(nil)
 
+type cassandraDBOption func(*cdb)
```

Comment:
nit: cmiw, this is a future-proof type and not actively used right now. Maybe a comment explaining this type would be nice.

Reply:
used right below as constructor parameter. It's a common pattern but let me add some comments here to make it obvious